py/desisim/scripts/quickquasars.py

```diff
@@ -248,8 +248,7 @@ def simulate_one_healpix(ifilename,args,model,obsconditions,decam_and_wise_filte
     # Read transmission from files. It might include DLA information, and it
     # might add metal transmission as well (from the HDU file).
     log.info("Read transmission file {}".format(ifilename))
-    trans_wave, transmission, metadata, dla_info = read_lya_skewers(ifilename,read_dlas=(args.dla=='file'),add_metals=args.metals_from_file)
-
+    trans_wave, transmission, metadata, dla_info = read_lya_skewers(ifilename,read_dlas=(args.dla=='file'),add_metals=args.metals_from_file)   
```

Comment:
What is the change for this line? and why add a space?

Reply:
No change here, might be a mistake in my editor.

Reply:
ok, could you revert? Thanks.

Reply:
@alxogm, it is because you have trailing spaces at the end of the line.